config/locales/views/schools/schools.yml

```diff
@@ -689,6 +689,9 @@ en:
     your_school_estates:
       edit:
         alternative_heating_sources:
+          air_source_heat_pump:
+            label: Our school has an air source heat pump
+            percentage: What percentage of the school using the air source heat pump for heating?
```

Comment:
I am not sure this actually makes sense, should it be 'is using'? I decided to use this sentence structure to be the same as the other alternative heating source fields. Which perhaps means these don't read correctly either? 🤔

Also the other ones capitalise the heating source, which also doesn't always seem right.